livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/dalle3.py

```diff
@@ -1,66 +0,0 @@
-# Copyright 2023 LiveKit, Inc.
```

Comment:
deleting this would break agent quickstart guide demo. any reasons to remove this?

I guess it'll be fine to put this as an example somewhere too in order to show people how to take the output and creating a VideoFrame out of it

Reply:
yeah we talked about this being an example instead of in the plugin (which i'm in favor of). Good callout re: quickstart.